### PR TITLE
Moving all shell scripts into bin directory

### PR DESCRIPTION
**Pull request by guineveresaenger**
Changed all references to these file locations

### CODE REVIEW

**Comment by guineveresaenger** [comment]
Addresses #525 .

**Comment by guineveresaenger** [comment]
Review: check if all file paths and locations make sense, including the README instructions. I successfully manually ran a cluster up and down with the new filepath.

WARNING: this will break k2cli, which is why I filed an issue [here](https://github.com/samsung-cnct/k2cli/issues/124).

File: docker/down.sh [hunk 1]
```diff
@@ -1,3 +0,0 @@
-#!/bin/bash
```
Comment (coffeepac) [inline docker/down.sh]: this file is required, please put it back

File: docker/up.sh [hunk 2]
```diff
@@ -1,3 +0,0 @@
-#!/bin/bash
```
Comment (coffeepac) [inline docker/up.sh]: this file is required, please put it back

File: docker/Dockerfile [hunk 3]
```diff
@@ -11,4 +11,4 @@ RUN 	apk update && \
        apk del git
 
 WORKDIR "/kraken"
-CMD ["./up.sh"]
\ No newline at end of file
+CMD ["./bin/up.sh"]
```
Comment (coffeepac) [inline docker/Dockerfile]: probably best to leave this one for now.  there is another issue, #639, for rebuilding the dockerfile.

File: docker/up.sh [hunk 4]
```diff
@@ -1,3 +0,0 @@
...
-cd /kraken
```
Comment (coffeepac) [inline docker/up.sh]: this should be changed to `cd /kraken/bin`

File: docker/down.sh [hunk 5]
```diff
@@ -1,3 +0,0 @@
...
-cd /kraken
```
Comment (coffeepac) [inline docker/down.sh]: this should be changed to `cd /kraken/bin`

File: README.md [hunk 6]
```diff
@@ -437,7 +437,7 @@ clusters:
 - etcd settings (beyond machine type)
 
 *Warning*
-Repeatedly running `./up.sh` or `./update.sh` can result in etcd nodes being destroyed and regenerated automatically, which will break your cluster's control plane. This is because in the K2 generated config file, the CoreOS version is set to `current`. When CoreOS releases a new version, Terraform will register this change and destroy and re-create etcd nodes. We are working on fixing this, but please be aware that this is a possible issue.
+Repeatedly running `up.sh` or `update.sh` can result in etcd nodes being destroyed and regenerated automatically, which will break your cluster's control plane. This is because in the K2 generated config file, the CoreOS version is set to `current`. When CoreOS releases a new version, Terraform will register this change and destroy and re-create etcd nodes. We are working on fixing this, but please be aware that this is a possible issue.
```
Comment (coffeepac) [inline README.md]: I think we can remove this warning now.  The work @davidewatson did for etcd prevents accidental etcd node rebuilds.

**Comment by guineveresaenger** [comment]
Additional Review criteria: 
Related PR for k2cli: https://github.com/samsung-cnct/k2cli/pull/126
from k2cli issue https://github.com/samsung-cnct/k2cli/issues/124

This was closely related, so I went ahead and fixed it.

**Pull request merged**